src/admin/create-cohort/create-cohort.js

```diff
@@ -1,10 +1,11 @@
 import React from "react";
+import CreateForm from "./create-form"
 
 const CreateCohort = () => {
   return (
     <>
       <h1>Create New Cohort Application Form</h1>
-      <p>Place holder for form</p>
+      <p><CreateForm /></p>
```

Comment:
I think this is causing a console error.  If we just remove the p tags this should fix it.